### DISCUSSION TITLE
How do you actually enable github packages?

### DISCUSSION THREAD

**Discussion by StephenDillon** [community/community#48731] [Packages]
### Select Topic Area

Product Feedback

### Body

I am getting a 403 Forbidden pushing a package. I assume it is because I never enabled / created anything before attempting to push, but when I got to the package page, it asks to select your registry but gives no ability to actually select one?
Just links to help pages, any idea how to actually select container registry?

![image](https://user-images.githubusercontent.com/1057182/221676196-d9da5c8e-8cb1-4f4f-9e1e-7777a95be538.png)

**Comment by almerindo** [comment]
Give a look at the [Quick Start Tutorial](https://docs.github.com/pt/packages/quickstart) 
Have you ever tried this?
